pkg/app/app.go

```diff
@@ -27,18 +25,26 @@ func initConfiguration(config configuration.Configuration, apiClient api.ApiClie
 	config.AddDefaultValue(configuration.CACHE_PATH, configuration.StandardDefaultValueFunction(dir))
 
 	config.AddDefaultValue(configuration.API_URL, func(existingValue any) any {
-		if existingValue == nil {
-			return constants.SNYK_DEFAULT_API_URL
-		} else {
-			apiString := existingValue
+		urlString := constants.SNYK_DEFAULT_API_URL
+
+		if existingValue != nil {
 			if temp, ok := existingValue.(string); ok {
-				if apiUrl, err := url.Parse(temp); err == nil {
-					apiUrl.Path = strings.Replace(apiUrl.Path, "/v1", "", 1)
-					apiString = apiUrl.String()
-				}
+				urlString = temp
 			}
-			return apiString
 		}
+
+		apiString, _ := api.GetCanonicalApiUrl(urlString)
+		return apiString
+	})
+
+	config.AddDefaultValue(configuration.WEB_APP_URL, func(existingValue any) any {
+		canonicalApiUrl := config.GetString(configuration.API_URL)
+		appUrl, _ := api.DeriveAppUrl(canonicalApiUrl)
+		return appUrl
+	})
+
+	config.AddDefaultValue(configuration.OAuthTokenUrl, func(existingValue any) any {
+		return "https://app.fedramp-alpha.snykgov.io"
```

Comment:
Most likely, this should be a constant

Reply:
I think the configuration value "OAuthTokenUrl" is obsolete, since it can be derived from the web_app_url. It used to be temporarily hardcoded in the oauthauthenticator since the api doesn't support the correct endpoints yet. but there is no need to have an extra config value for it.

Reply:
It was hard coded, but I wanted a way to control it in the tests so I added a configuration